apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx

```diff
@@ -155,28 +154,26 @@ const TemplateEditorFormContext = createContext<ITemplateEditorFormContext>({
   deleteVariant: () => {},
 });
 
-const defaultValues: IForm = {
-  name: 'Untitled',
-  notificationGroupId: '',
-  description: '',
-  identifier: '',
-  tags: [],
-  critical: true,
-  steps: [],
-  preferenceSettings: {
-    email: true,
-    sms: true,
-    in_app: true,
-    chat: true,
-    push: true,
-  },
-};
-
 const TemplateEditorFormProvider = ({ children }) => {
   const { templateId = '' } = useParams<{ templateId?: string }>();
   const methods = useForm<IForm>({
     resolver: zodResolver(schema as any),
-    defaultValues,
+    defaultValues: {
+      name: 'Untitled',
+      notificationGroupId: '',
```

Comment:
try to revert in a separate commit.

Reply:
Reminder to see if there are any action items required for this